environments/production/known_hosts

```diff
@@ -1,152 +1,205 @@
 172.24.32.28 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBFDfebBOcQHMYiQySNA+bGmksSRabcvVZPQgYu8MzdIAt7mbZ11JI3irhYC/oEm99r/m1lNb3O8KhvoTAeylCmM=
+172.24.32.65, ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBMMJFVOAJe3F1sFB23PUvJue6zNqChq7TBdWVlJLZ1WVtLIy7ON5uE6eNeoEhCvZXYw/kE9+TmVDZTaQvEqnjmA=
+172.24.32.65, ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDxIBTrizHHjJSTct81cv63PRtJn2sj20cmoHVaUjfyB7j3/RYRmU54iP/X8X7/kawT5RKraXDp9BUBuKIPgD9AjAJAfY1SMCugty3cP0aKlIwpELnRQrPJ9gbxtG0lriGPiKEI34PcmXs0WpTvKdNKSUsAbrsd1ryddfDcCZO//mctxg9nsOkvVxwH7WCNMoU38WX+/WjjafYGDnNPYk20sbD9RRCH1cRiHXco1bsUswJqX16t4tVAtxK5x876UBM5p6xFzhtpMq6HlZCWxZjSXN5FAAB8b4KuAfaEWSPjqqLXIwTHMmvNejV+HDzqkokGZMFzo7yYxJV+lD7sNfWL
+172.24.32.66, ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBN4ps8+QhwJ+cRyzxh0wpBOGPMJor0dUw1joyc3OMn5Og2XSjK9lJdhLr7bInjtdUuexFEpTXR3OWtbHCBfSXHg=
+172.24.32.66, ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC2OToWTpkqmVdkPhPJLyOh3+A/ooCLnTfNNl7xBIJMPz2SKrezGXYT/SNRZMVSnwXXxQnlMjogK7D31ugvjE5v9XuWVCYFxnVAtEhio0ZTdULhyyZmYmYhiYvzAKwUkZM1Rm4z4c80+MLPaBP3+Dj6E/NOOTsFqNWF0tg/iHuTWXuKgjcr2PAfiW4B1i+UDse3V31esPMyg0AVdDEhODtcV+AwCu21aJZ00y4leEhn1GImNtmDwRDiUjuI45OsCX4fqp4QiMLSnuxF7RvlGpEKZq21VVgPItmaKoK9K/EuUNWG1nKhU81wQ3+f3mPdyJmewVMe1gdYRcagEKOHnqLj
+control.internal-va.commcarehq.org,172.24.32.28 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBFDfebBOcQHMYiQySNA+bGmksSRabcvVZPQgYu8MzdIAt7mbZ11JI3irhYC/oEm99r/m1lNb3O8KhvoTAeylCmM=
```

Comment:
Is it okay that this changed things other than the two es machines?

also, 
did you just run `update-local-known-hosts` on local machine for this?